Scripts/libs_utils.py

```diff
@@ -0,0 +1,226 @@
+import os, sys, shutil, re
+from subprocess import check_output
+
+# this script forces to link libraries again on next IDE run
+# (inside present a workaround for build.cs file, that's trick is to avoid deleting 'intermediate' folder at all)
+def InvalidateBinaries(turtlebot3Binaries, rclUEBinaries, rclUEBuildCS):
+    cwd = os.getcwd()
+
+    if os.path.isdir(turtlebot3Binaries):
```

Comment:
Seems like this could just be projectBinaries, since it is not specific to Turtlebot.

Reply:
Yeah, I understand. Will change. all this paths I left, just for the latest study of my refactor